docs/dev/stream/operators/process_function.md

```diff
@@ -246,10 +246,8 @@ class CountWithTimeoutFunction extends KeyedProcessFunction[Tuple, (String, Stri
 </div>
 </div>
 
-{% top %}
-
 
-**NOTE:** Before Flink 1.4.0, when called from a processing-time timer, the `ProcessFunction.onTimer()` method sets
+<span class="label label-info">Note</span> Before Flink 1.4.0, when called from a processing-time timer, the `ProcessFunction.onTimer()` method sets
```

Comment:
I think `**NOTE:**` is used in more places?

Reply:
in process_function.md，`<span class="label label-info">Note</span>`  is used in more places，but `**NOTE:**` is used only place

Reply:
> I think `**NOTE:**` is used in more places?
`**NOTE:**` is used only one place in process_function.md,  
and `<span class="label label-info">Note</span>` is used in more places.

Reply:
got.